Protocol sync updates (#394)
MukurtuCMS/mukurtu-user-manual: docs/media/ManageMediaAccessWithProtocols.md

!!! tip 
    In Mukurtu v3 protocols were optional. This is a new requirement for Mukurtu v4.

> In most cases you will want to apply the same cultural protocols to each media asset that are applied to the content items where the media asset is used. This ensures that if a user can view the content, they can also view the media assets. To this end, we have included an optional **Sync protocols with content** setting. When enabled, this setting ensures the media asset's protocols will always match those of it's parent content to keep content and media access in sync. Media created within a content form will automatically inherit content protocols. Media created elsewhere require protocol selection, but will sync once added to content. You can disable the sync setting to set media protocols independent of parent content. 

Reviewer: ```suggestion
In most cases you will want to apply the same cultural protocols to each media asset that are applied to the content items where the media asset is used. This ensures that if a user can view the content, they can also view the media assets. This is facilitated by the **Sync protocols with content** setting. When enabled, this setting ensures the media asset's protocols will always match those of it's parent content to keep content and media access in sync. Media with synced protocols created within a content form will automatically inherit content protocols. Media created elsewhere require protocol selection, but will sync once added to content. You can disable the sync setting to set media protocols independent of parent content. 
```

When the **Sync protocols with content** toggle is selected, the cultural protocol and sharing settings options are automatically hidden.

![Screenshot of the cultural protocols field with a protocol selected and the sync protocols with content field highlighted.](../_embeds/sync2.png)

![Screenshot of the cultural protocols field with the sync protocols with content toggle selected.](../_embeds/sync1.png)

You can have different protocols between content and media if this is appropriate or preferred for your site. This allows you to further manage access to media assets for your site users while keeping the metadata more widely accessible. For example, a digital heritage item under an open protocol featuring a media asset under a strict protocol would make the metadata and item discoverable, but limit access to the media assets.

> ![Screenshot of the menu view of a digital heritage item with an open protocol featuring a media asset with a strict cultural protocol.](../_embeds/mediaadminreview.png)

Reviewer: These last two screenshots don't really show anything useful. Maybe one of the edit form showing a media asset and one of the public view.
